src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -16,12 +16,7 @@
 package com.amazon.dlic.auth.http.jwt;
 
 import java.nio.file.Path;
-import java.security.AccessController;
-import java.security.Key;
-import java.security.KeyFactory;
-import java.security.NoSuchAlgorithmException;
-import java.security.PrivilegedAction;
-import java.security.PublicKey;
+import java.security.*;
```

Comment:
we should avoid *, I think intellij do this by default when import from same packages increase by certain limit.

Reply:
yes, I didn't realize this, seems that intellij did this. will edit